frontend/src/screens/Home/index.tsx

```diff
@@ -1,18 +1,140 @@
+import { CardLocation } from '~/components/Cards/CardLocation'
+import { dataForYou, dataOutstanding, dataLocation, dataNews } from '~/data/fakeData'
+import Sliders from '~/Others/Slider'
 const Home = () => {
   return (
-    <div>
-      <div className="relative">
-        <img src={banner} alt="banner" className="w-full overflow-hidden" />
-        <form className="absolute left-1/2 top-1/2 flex w-[70%] -translate-x-1/2 -translate-y-1/2 transform items-center rounded-lg border-2 border-gray-primary bg-white pl-3">
-          <Input type="text" placeholder="Tìm nhanh. VD: Vinhomes Cenntral Park" className="w-full indent-1" />
-          <Button size="sm" variant="primary" className="w-40 sm:w-28">
-            Tìm kiếm
-          </Button>
-        </form>
+    <div className="m-0 flex h-[880px] flex-col overflow-y-auto p-0">
+      <div className="">
+        <div className="relative">
+          <img src={banner} alt="banner" className="w-full overflow-hidden" />
+          <form className="absolute left-1/2 top-1/2 flex w-[70%] -translate-x-1/2 -translate-y-1/2 transform items-center rounded-lg border-2 border-gray-primary bg-white pl-3">
+            <Input type="text" placeholder="Tìm nhanh. VD: Vinhomes Cenntral Park" className="w-full indent-1" />
+            <Button size="sm" variant="primary" className="w-40 sm:w-28">
+              Tìm kiếm
+            </Button>
+          </form>
+        </div>
+
+        <div className="flex flex-col overflow-hidden bg-[--gray-secondary] p-5 pb-10 sm:w-auto xl:px-[15%]">
+          <div className="mb-10 flex justify-between py-5 sm:items-center">
+            <h1 className="text-xl font-medium sm:text-3xl">Dành cho bạn</h1>
+            <Link to={''} className="flex h-full">
+              <p className="mr-2 text-base">Xem tất cả</p>
+              <img src={expandTopRight} alt="expand top right" />
+            </Link>
+          </div>
+          <div className="m-5 overflow-visible">
+            <Sliders>
+              {dataForYou.map((item, index) => (
+                <div className="p-2">
+                  <Link to="">
+                    <CardReport
+                      key={index}
+                      imageUrls={item.img}
+                      name={item.title}
+                      regularPrice={item.price}
+                      area={item.area}
+                      address={item.address}
+                      date={item.date}
+                      save={item.vote}
+                      isOpen
+                      isOutStanding={false}
+                    />
+                  </Link>
+                </div>
+              ))}
+            </Sliders>
+          </div>
+        </div>
+
+        <div className="flex flex-col overflow-hidden bg-white p-5 pb-10 sm:w-auto xl:px-[15%]">
+          <div className="mb-10 flex items-center justify-center">
+            <h1 className="py-5 text-base font-medium sm:text-3xl">Bất động sản theo địa điểm</h1>
```

Comment:
H1 dùng để làm tiêu đề to của website thôi, không dùng để làm tiêu đề content
```suggestion
            <h3 className="py-5 text-base font-medium sm:text-3xl">Bất động sản theo địa điểm</h1>
```